tests/ips-state-actions-test-b-01/test.yaml

```diff
@@ -0,0 +1,30 @@
+pcap: ../tls/tls-certs-alert/input.pcap
+
+args:
+  - --set stream.midstream=false
+  - -k none
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: tls
+  - filter:
+      min-version: 7
```

Comment:
This is a behavior change between 6 and 7+master. I think it makes sense, as the app_proto can change.